uplink/storage/streams/store.go

```diff
@@ -153,34 +152,40 @@ func (s *streamStore) upload(ctx context.Context, path Path, pathCipher storj.Ci
 		}
 	}()
 
-	eofReader := NewEOFReader(data)
+	var (
```

Comment:
maybe we can move variables from inside loop to the loop and what is from method scope move to previous `var` block?

Reply:
I may miss some of these variables, but these variables are declared outside of the loop because are needed after the loop finishes for holding the value of the last iteration.

On the other hand, I kept the `eofReader := NewEOFReader(data)` (of the line 165) outside of this `var` block because I thought that it was clearer that what it iterating is just declared before the loop starts. If you think that `eofReader` is better to be in the `var` block I'm happy to move it.

Let me know if you find any variable of this block which is only used inside of the loop and also if you'd like to move the `eofRedader` declaration.

Reply:
sorry I think I was wrong. I tough that more variables are only for loop but it looks that its only `contentNonce` and I think it will be better to keep it next to `contentKey`.

Reply:
Oh, good.

I prefer to move `contentNonce` declaration inside of the loop as it's only used there despite the close relation with `contentKey`